### PR TITLE
Add OpenH264 support and dependency to GStreamer Plugins Bad 1.6.3, RabbitMQ C Client, libsrtp, usrsctp, sofia-sip, libmicrohttpd fix

### PR DESCRIPTION
**Pull request by marcin-sielski**

### CODE REVIEW

File: mingw-w64-rabbitmq-c/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+_realname=rabbitmq-c
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=0.8.0
+pkgrel=1
+pkgdesc="RabbitMQ C client (mingw-w64)"
+arch=('any')
+url='https://github.com/alanxz/rabbitmq-c'
+license=('MIT')
+validpgpkeys=('gpg_KEY')
```
Comment (mati865) [inline mingw-w64-rabbitmq-c/PKGBUILD]: Either use valid GPG key or remore `validpgpkeys`.

File: mingw-w64-rabbitmq-c/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake" "${MINGW_PACKAGE_PREFIX}-gcc")
+depends=("${MINGW_PACKAGE_PREFIX}-openssl" "${MINGW_PACKAGE_PREFIX}-popt")
+options=('strip' 'staticlibs')
+source=("${_realname}-${pkgver}.tar.gz"::"https://github.com/alanxz/${_realname}/archive/v${pkgver}.tar.gz"
+        "0001-librabbitmq-CMakeLists.txt.patch")
+sha256sums=('SKIP'
+            'SKIP')
```
Comment (mati865) [inline mingw-w64-rabbitmq-c/PKGBUILD]: Checksums shouldn't be skipped unless source is being downloaded from VCS. Use `updpkgsums` in directory with the PKGBUILD to generate them.

File: mingw-w64-openh264/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+_realname=openh264
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.4.0
+pkgrel=1
+pkgdesc="Library for encoding/decoding H264/AVC video streams (mingw-w64)"
+arch=('any')
+url="http://www.openh264.org/"
+license=("BSD")
+validpgpkeys=('gpg_KEY')
```
Comment (mati865) [inline mingw-w64-openh264/PKGBUILD]: Either use valid GPG key or remore `validpgpkeys`.

File: mingw-w64-openh264/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc" "${MINGW_PACKAGE_PREFIX}-pkg-config" "${MINGW_PACKAGE_PREFIX}-nasm")
+depends=()
+options=('strip' 'staticlibs')
+source=("${_realname}-${pkgver}.tar.gz"::"https://github.com/cisco/openh264/archive/v${pkgver}.tar.gz")
+sha256sums=('SKIP')
```
Comment (mati865) [inline mingw-w64-openh264/PKGBUILD]: Checksums shouldn't be skipped unless source is being downloaded from VCS. Use `updpkgsums` in directory with the PKGBUILD to generate them.

**Comment by marcin-sielski** [comment]
Fixed

**Pull request merged**